app/lib/l10n/app_en.arb

```diff
@@ -188,5 +188,7 @@
       "placeholders": {
       "pattern": {}
       }
-  }
+  },
+  "app_outdated_message": "A new version of the StudyU app is available. Please update it to receive the latest features and improvements. Thank you for your support!",
+  "open_store": "Open Store"
```

Comment:
If we don't want to refer to the actual store names to ease localization, let's phrase this differently since "Open Store" could be perceived as a specific term. Better use something like "Update now".